app/javascript/components/App.js

```diff
@@ -37,6 +38,21 @@ class App extends React.Component {
   }
 
 
+  //create a fetch
+  createApartment = (newApartment) => {
+    fetch("http://localhost:3000/apartments", {
```

Comment:
Your fetch should be to just `"/apartments"` since you are already on localhost.